tests/special_coverage/test_spec_cov_views.py

```diff
@@ -0,0 +1,109 @@
+from django.contrib.auth import get_user_model
+from django.core.urlresolvers import reverse
+from django.test import Client, override_settings
+from django.utils import timezone
+
+from bulbs.utils.test import BaseIndexableTestCase, make_content
+from bulbs.campaigns.models import Campaign
+from bulbs.special_coverage.models import SpecialCoverage
+
+
+class TestSpecialCoverageViews(BaseIndexableTestCase):
+
+    def setUp(self):
+        super(TestSpecialCoverageViews, self).setUp()
+        User = get_user_model()
+        self.client = Client()
+
+        admin = self.admin = User.objects.create_user("admin", "tech@theonion.com", "secret")
+        admin.is_staff = True
+        admin.save()
+
+        self.campaign = Campaign.objects.create(
+            sponsor_name="Campaign"
+        )
+
+    def test_special_coverage_view(self):
+        content = make_content(published=timezone.now())
+        content.__class__.search_objects.refresh()
+
+        sc = SpecialCoverage.objects.create(
+            name="Test Coverage",
+            slug="test-coverage",
+            description="Testing special coverage",
+            query={
+                "included_ids": [content.id]
+            },
+            videos=[],
+            campaign=self.campaign,
+            start_date=timezone.now() - timezone.timedelta(days=10),
+            end_date=timezone.now() + timezone.timedelta(days=10)
+        )
+        sc.save()
+
+        response = self.client.get(reverse("special", kwargs={"slug": sc.slug}))
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.context['content_list'].count(), sc.get_content().count())
+        self.assertEqual(response.context['content_list'][0].id, content.id)
+
+    def test_inactive_special_coverage_view(self):
+        content = make_content(published=timezone.now())
+        content.__class__.search_objects.refresh()
+
+        # create old special coverage
+        sc = SpecialCoverage.objects.create(
+            name="Test Coverage",
+            slug="test-coverage",
+            description="Testing special coverage",
+            query={
+                "included_ids": [content.id]
+            },
+            videos=[],
+            campaign=self.campaign,
+            start_date=timezone.now() - timezone.timedelta(days=20),
+            end_date=timezone.now() - timezone.timedelta(days=10)
+        )
+        sc.save()
```

Comment:
we actually don't need this either. i was running into some unrelated elasticsearch issues and forgot to take this out.